app/controllers/v0/claim_letters_controller.rb

```diff
@@ -34,8 +34,9 @@ def service
     def allowed_doctypes
       doctypes = %w[184]
       doctypes << '27' if Flipper.enabled?(:cst_include_ddl_boa_letters, @current_user)
-      doctypes << '65' if Flipper.enabled?(:cst_include_ddl_5103_letters, @current_user)
-      doctypes << '68' if Flipper.enabled?(:cst_include_ddl_5103_letters, @current_user)
+      doctypes << '704' if Flipper.enabled?(:cst_include_ddl_5103_letters, @current_user)
```

Comment:
Im confused on why we are removing 65 and 68 and adding 704, 706, 858. If you could add some information about this to your ticket that'd be really helpful! I took a look at the link you provided [here](http://vets.gov/).[user+227@gmail.com](mailto:user+227@gmail.com) and it looks to me like we should be adding 704, 705 , 706 and 707 so im very confused. Thanks!

Reply:
Talked to @jacobworrell in parking lot 

- letter id 704 correlates to 65 and we were just referencing the wrong id#. It is for **Standard 5103 Notice**
- letter id 706 correlates to 68 and we were just referencing the wrong id#. It is for **5103/ DTA letter**
- letter id 858 needed to be added and is for **Custom 5103 Notice**